redis_mimic.py

```diff
@@ -0,0 +1,26 @@
+import random
+import time
+printers = ('printer-logjam', 'printer-pagefault', 'printer-papercut')
+
+def randUsername():
+	username = ''
```

Comment:
here and throughout this file: please use four spaces instead of tab characters for indentation